### PR TITLE
Separate utils

### LINKED ISSUE #196: Setup install requirements include libraries only used in utils folder
Related discussion: #195 

Should we move the `utils` folder to its own "side" package? Not everyone will be using the parsers that we have provided, and this might result in them installing packages they don't need along with Dash Bio.

### PR DESCRIPTION
**Pull request by shammamah**
Closes #196 

Related to https://github.com/plotly/dash-bio-utils/pull/1

## About
- [ ] This is a new component
- [ ] I am adding a feature to an existing component, or improving an existing feature
- [x] I am closing an issue

## Description of changes
- remove `utils/` folder
- use external `dash_bio_utils` package for all apps

## Before merging
- [ ] I have gone through the [code review checklist](https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/review_checklist.md)
- [ ] I have completed all of the [steps to take before merging](https://github.com/plotly/dash-bio/blob/master/.github/before_merging.md)
- [ ] I know how to [deploy to DDS](https://github.com/plotly/dash-bio/blob/master/.github/deploying_to_dds.md)

### CODE REVIEW

**Comment by mkcor** [comment]
I guess we need that https://github.com/plotly/dash-bio-utils/pull/1 merged in before shipping this one.

**Comment by mkcor** [comment]
Not directly or specifically related to this PR, but the test failure which happened upon 7f642cd (had to rerun the CI workflow for it to pass) is a failure I've observed locally:
ideogram_assert_error
In the past, I wasn't able to 'reproduce' locally all these failures which happen intermittently with *Ideogram* tests. So, now I'm thinking of going through them again to make them more reliable...

**Comment by shammamah** [comment]
@mkcor Yes, I think it would be a good idea to clean up the tests again as well. I think we should focus on developing tests now. Ideally we would have been doing test-driven development this whole time, but now that the package is released it gives us some time to catch up before we start working on adding new features.

**Comment by mkcor** [comment]
@shammamah I'm really only talking about the Ideogram tests here, because they've been failing repeatedly, for some CSS elements can't be found within the default timeout limit. Happy to write more/new tests of course, but I was thinking of fixing the Ideogram thing first.

File: dash_bio/package-info.json [hunk 1]
```diff
@@ -1 +1 @@
-{"name": "dash_bio", "version": "0.0.11", "author": "The Plotly Team"}
\ No newline at end of file
+{"name": "dash_bio", "version": "0.1.0rc2", "author": "The Plotly Team"}
```
Comment (mkcor) [inline dash_bio/package-info.json]: @shammamah I'm not sure I've seen version `0.1.0rc1` -- is there another PR I should be approving before this one?

File: requirements.txt [hunk 2]
```diff
@@ -4,7 +4,8 @@ biopython==1.73
 colour==0.1.5
 cython>=0.19
 dash==0.40.0
-dash-bio==0.0.11
+dash-bio==0.1.0rc2
+dash-bio-utils==0.0.1
```
Comment (mkcor) [inline requirements.txt]: Yay! :)

File: dash_bio/package-info.json (hunk 1 above)
Comment (shammamah) [reply dash_bio/package-info.json]: Nope! I was just testing stuff out.

Comment (shammamah) [reply dash_bio/package-info.json]: It was in this PR: d62dbcd1f77a41b23ca35f516801ad04ca341b57

Comment (mkcor) [reply dash_bio/package-info.json]: That's a commit, not a PR... even inspecting the commit history around May 16th (date of commit d62dbcd1f77a41b23ca35f516801ad04ca341b57), I can't figure out which PR it was submitted with, but anyhow.

**Pull request merged**